create-liff-app.ts

```diff
@@ -47,40 +29,25 @@ export function init() {
   console.log(
     `${chalk.greenBright('Welcome')} to the ${chalk.cyan('Create LIFF App')}`
   );
-  prompt(questions).then(async (answers) => await createLiffApp(answers));  
+  prompt(questions).then(async (answers) => await createLiffApp(answers));
 }
 
 type PackageManager = 'npm' | 'yarn'
 
 export async function createLiffApp(answers: Answers) {
-  const { projectName, template, language, installNow, liffId } = answers;
-  const packageManager: PackageManager = installNow ? await inquirer.prompt({
-    type: 'list',
-    name: 'packageManager',
-    message: 'Which package manager do you want to use?',
-    choices: [
-      {
-        key: 'yarn',
-        value: 'yarn',
-        checked: true,
-      },
-      {
-        key: 'npm',
-        value: 'npm',
-        checked: false,
-      },
-    ],
-  }).then(({ packageManager }) => packageManager as PackageManager) : 'npm';
-  
-  const templateName = `${template}${language === 'JavaScript' ? '' : '-ts'}`;
+  const { projectName, template, language, liffId } = answers;
+  const templateConfig = templates[template];
```

Comment:
Sorry for nitpicking but let's make more type-safe

```suggestion
  const templateConfig = templates[template] as TemplateOptions | undefined;
  if (!templateConfig) {
    throw new Error(`Invalid template name: ${template}`);
  }
```

Reply:
of course!